.github/workflows/playwright.yml

```diff
@@ -49,7 +49,7 @@ jobs:
     needs: [build-plugin]
     strategy:
       matrix:
-        testSuite: ['default', 'nested-tabs', 'reverse-columns']
+        testSuite: ['elements-regression', 'default', 'nested-tabs', 'reverse-columns']
```

Comment:
if possible lets keep the default the 1st item

Reply:
I did this on purpose - `elements-regression` is the only one here not in the "usual" framework, so I wanted it to stand out.
If I had placed it last, the next test would come after it.
Another option, would be: `['default', 'elements-regression', 'nested-tabs', 'reverse-columns']`

What do you think?